pkg/pool-manager/virtualmachine_pool.go

```diff
@@ -314,7 +314,7 @@ func (p *PoolManager) initVirtualMachineMap() error {
 		}
 	}
 
-	return nil
+	return p.UpdateWaitingTimeOnInit(waitingMac)
```

Comment:
Commit message comment:
That is one long sentece. Please exaplain when does it happen, what happens, what is the consequence.